x-pack/legacy/plugins/spaces/index.ts

```diff
@@ -86,38 +82,7 @@ export const spaces = (kibana: Record<string, any>) =>
           hidden: true,
         },
       },
-      home: ['plugins/spaces/register_feature'],
-      injectDefaultVars(server: any) {
-        return {
-          spaces: [],
-          activeSpace: null,
```

Comment:
No more `activeSpace` means that the client-side must fetch this on demand. I've moved this functionality to the `SpacesManager`, which fetches it once, and caches it for its lifetime.

Reply:
Good catch! I've restored this variable solely for their use in https://github.com/elastic/kibana/pull/40856/commits/0186d5f528f798b1ccc67ac488ae2de37cc05f15